scripts/parallellize_cargo_check.py

```diff
@@ -36,6 +36,20 @@
 ]:
     print("Running ", task)
     print(os.environ)
+    if (("--manifest-path libafl_qemu/Cargo.toml" in task or "--manifest-path libafl_qemu/libafl_qemu_sys/Cargo.toml" in task)
```

Comment:
not a huge fan of this one, it's not very readable imho

Reply:
better now?

Reply:
yes, looks good